src/bin/cli.ts

```diff
@@ -25,6 +29,7 @@ program
     '--baseUrl <baseUrl>',
     'Change the base url for github request. For example if used on a Github Enterprise Server instance.'
   )
+  .option('-r, --repo <name>', 'Allow access to a single repository', collectRepos, [])
```

Comment:
```suggestion
  .option('-r, --repo <name>', 'Allow access to a single repository', (value: string, previous: string[]) => [...previous, value], [])
```

Reply:
This is another option. Given that is a one liner, written this way can be easier to just leave it there in the option.